src/dependency-manager/src/service-config/v1.ts

```diff
@@ -45,7 +54,16 @@ export class ServiceConfigV1 extends ServiceConfig {
   debug?: string;
   parameters: { [s: string]: ServiceParameterV1 } = {};
   datastores: { [s: string]: ServiceDatastoreV1 } = {};
-  api: ApiSpecV1 = { type: 'rest' };
+  api: ApiSpecV1 = {
+    type: 'rest',
+    liveness_probe: {
+      path: '/health',
```

Comment:
I don't think we want defaults/force them to configure correctly. Make liveness_probe optional.

Reply:
Also make every key in the interfaces optional `?` except `path`. Figure we can have the defaults server-side if they don't fill everything out.